### DISCUSSION TITLE
How to vertical align center?

### DISCUSSION THREAD

**Discussion by tz301** [zauberzeug/nicegui#3174] [Q&A] [answered]
### Question

```py
with ui.row():
    ui.button('Press')
    ui.input('Input')
```

How to make these two container align vertical center?

![image](https://github.com/zauberzeug/nicegui/assets/35760938/d565686d-1af4-43ac-ad30-6a4d63aeec33)

**Comment by falkoschindler** [comment, marked as answer]
Hi @tz301,

You can use Tailwind's "items-center" class:
```py
with ui.row().classes('items-center'):
    ui.button('Press')
    ui.input('Input')
```
<img width="253" alt="Screenshot 2024-06-06 at 07 30 03" src="https://github.com/zauberzeug/nicegui/assets/5767091/afd60ff6-61c9-410f-a0f6-304db380bc8e">

Since this is a very common use case, I just created PR #3175 to introduce a dedicated parameter `align_items`, which will part of release 1.4.27.

**Reply by tz301** [reply]
Thanks!
